src/utils/firebase.js

```diff
@@ -2,21 +2,29 @@ import firebase from 'firebase/app'
 import 'firebase/auth'
 import { removeUserInfo } from '@/utils/auth'
 import store from '../store/index'
+import { firebaseConfig } from '/config/firebaseConfig'
 
 class FirebaseAuth {
   constructor() {
+    try {
+      this.app = firebase.initializeApp(firebaseConfig)
+    } catch (error) {
+      this.app = null
+      console.error(error)
+    }
+
```

Comment:
Looks like the "firebase app" will be initialized whenever the `FirebaseAuth` object is instantiated.
Just a quick question, so when *exactly* is this class instantiated?
Judging from line 161, a FirebaseAuth() object is created there.
So is the object created among import in `main.js`? (quick reference [here](https://github.com/NCTU-SYNC/sync-frontend/blob/55408524a6438c5cf584747a0e85d5cbe4a19910/src/main.js#L9))
Thanks a lot!

Reply:
Noted.